index.html

```diff
@@ -68,10 +69,20 @@
 </body>
 
 <script>
+const SOUND_ENABLED_KEY = "soundEnabled"
+document.querySelector('#sound_toggle').checked = isSoundEnabled();
```

Comment:
Can you please move this `.checked` setter to line 245, so the consts can stay at the top and actions at the bottom?